packages/vue-app/template/client.js

```diff
@@ -14,8 +14,13 @@ import {
   compile,
   getQueryDiff,
   globalHandleError
-} from './utils'
-import { createApp, NuxtError } from './index'
+} from './utils.js'
+import { createApp, NuxtError } from './index.js'
```

Comment:
Is that change necessary?

Reply:
I do believe we will benefit by giving extension name for faster path resolution for the loader.